.gitmodules

```diff
@@ -71,7 +71,7 @@
 
 [submodule "Host.lwip"]
 	path = Sming/Arch/Host/Components/lwip/lwip
-	url = git://git.savannah.gnu.org/lwip.git
+	url = https://git.savannah.nongnu.org/git/lwip.git
```

Comment:
@mikee47 Why is that change here?

Reply:
#1988 refers. This same repo. is used as a submodule in esp8266 LWIP, using this URL. It didn't fix the problem so can pull this commit out and deal with that separately.

Reply:
> so can pull this commit out and deal with that separately.

Yes, please.